test/integration/projects.test.js

```diff
@@ -20,6 +20,7 @@ describe('Project Integration', function () {
     expect(projectResponse.data.mechanism).to.be.a('string');
     expect(projectResponse.data.latitude).to.be.a('number');
     expect(projectResponse.data.longitude).to.be.a('number');
+    expect(projectResponse.data.highlights).to.be.a('array');
```

Comment:
[nit] does 
```suggestion
    expect(projectResponse.data.highlights).to.be.an('array');
```
work?

Reply:
Yes it does!